### PR TITLE
Adding qualtircs source documentation

### PR DESCRIPTION
**Pull request by carlmlee**
### Proposed changes

Adding the initial Qualtrics source documentation

### Merge timing
To align with readiness of the source for public beta.

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md [hunk 1]
```diff
@@ -1,4 +1,56 @@
 ---
 title: 'Qualtrics Source'
 hidden: true
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/qualtrics/index.md:3]:
```suggestion
hidden: true
beta: true
```

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md [hunk 2]
```diff
@@ -1,4 +1,56 @@
...
----
\ No newline at end of file
+---
+
+---
+
+## title: Qualtrics Source
+
+[Qualtrics](https://qualtrics.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) [Qualtrics]() is an Experience Management platform that allows companies to design and improve customer and employee experiences through listening, analysis and action.
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/qualtrics/index.md:10]:
```suggestion
[Qualtrics](https://qualtrics.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is an Experience Management platform that allows companies to design and improve customer and employee experiences through listening, analysis, and action.
```

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md [hunk 3]
```diff
@@ -1,4 +1,56 @@
...
+
+This is an [Event Cloud Source](https://segment.com/docs/sources/#event-cloud-sources) which can not only export data into your Segment warehouse, but they can also federate the exported data into your other enabled Segment Destinations.
+
+This source is maintained by Qualtrics. For any issues with the source, [contact their Support team](mailto:support@qualtrics.com).
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/qualtrics/index.md:14]:
```suggestion
Qualtrics maintains this source. For any issues with the source, [contact the Qualtrics Support team](mailto:support@qualtrics.com).
```

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md [hunk 4]
```diff
@@ -1,4 +1,56 @@
...
+
+## Adding Destinations
+
+Now that your Source is set up, you can connect it with Destinations.
+
+Log into your downstream tools and check to see that your events appear as expected, and that they contain all of the properties you expect. If your events and properties don’t appear, check the [Event Delivery](https://segment.com/docs/connections/event-delivery/) tool, and refer to the Destination docs for each tool for troubleshooting.
+
+If there are any issues with how the events are arriving to Segment, [contact the Qualtrics support team](mailto:support@Qualtrics.com).
+
+---
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/qualtrics/index.md:56]:
```suggestion

```

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md [hunk 5]
```diff
@@ -1,4 +1,56 @@
...
+
+_**NOTE:** The Qualtrics Source is currently in beta, which means that they are still actively developing the source. This doc was last updated on February 15, 2023. If you are interested in joining their beta program or have any feedback to help improve the Qualtrics Source and its documentation, [let their team know](mailto:support@qualtrics.com)!_
+
+## Getting Started
+
+1. From your workspace's [Sources catalog page](https://app.segment.com/goto-my-workspace/sources/catalog) click **Add Source**.
+2. Search for "Qualtrics" in the Sources Catalog, select Qualtrics, and click **Add Source**.
+3. On the next screen, give the Source a nickname configure any other settings.
+
+   - The nickname is used as a label in the Segment app, and Segment creates a related schema name in your warehouse. The nickname can be anything, but we recommend using something that reflects the source itself and distinguishes amongst your environments (eg. SourceName_Prod, SourceName_Staging, SourceName_Dev).
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/qualtrics/index.md:24]:
```suggestion
   - The nickname is used as a label in the Segment app, and Segment creates a related schema name in your warehouse. The nickname can be anything, but Segment recommends using something that reflects the source itself and distinguishes amongst your environments (for example, SourceName_Prod, SourceName_Staging, or SourceName_Dev).
```

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md [hunk 6]
```diff
@@ -1,4 +1,56 @@
...
+This source is maintained by Qualtrics. For any issues with the source, [contact their Support team](mailto:support@qualtrics.com).
+
+_**NOTE:** The Qualtrics Source is currently in beta, which means that they are still actively developing the source. This doc was last updated on February 15, 2023. If you are interested in joining their beta program or have any feedback to help improve the Qualtrics Source and its documentation, [let their team know](mailto:support@qualtrics.com)!_
+
+## Getting Started
+
+1. From your workspace's [Sources catalog page](https://app.segment.com/goto-my-workspace/sources/catalog) click **Add Source**.
+2. Search for "Qualtrics" in the Sources Catalog, select Qualtrics, and click **Add Source**.
+3. On the next screen, give the Source a nickname configure any other settings.
+
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/qualtrics/index.md:23]:
```suggestion
```

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md [hunk 7]
```diff
@@ -1,4 +1,56 @@
...
+   - The nickname is used as a label in the Segment app, and Segment creates a related schema name in your warehouse. The nickname can be anything, but we recommend using something that reflects the source itself and distinguishes amongst your environments (eg. SourceName_Prod, SourceName_Staging, SourceName_Dev).
+
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/qualtrics/index.md:25]:
```suggestion
```

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md [hunk 8]
```diff
@@ -1,4 +1,56 @@
...
+4. Click **Add Source** to save your settings.
+5. Next login to your Qualtrics Account. Navigate to workflows -> Select the workflow you wish to send Segment events from -> Add a new Segment task.
+6. From within the Segment task, after connecting via your workspace API key, you now select your created Qualtrics source and continue to setup the task with the event, data mapping etc. 
+7. See more information on the Qualtrics Segment task at the following [Qualtrics support page](https://www.qualtrics.com/support/integrations/twilio-segment/twilio-segment-task/)
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/qualtrics/index.md:29]:
```suggestion
7. For more information on the Qualtrics Segment task, view the [Qualtrics support page](https://www.qualtrics.com/support/integrations/twilio-segment/twilio-segment-task/){:target="_blank"}.
```

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md [hunk 9]
```diff
@@ -1,4 +1,56 @@
...
+
+## Events
+
+The Qualtrics integration allows you to define the event to be sent to Segment from within the task. This can be customized to your particular use case e.g. 'Onboarding Survey Completed' could be the event based on a response to a particular survey. Another example may be 'Contact Updated' based on XM Directory change. These events can be tailored to align with your existing process or particular use case
+
+## Event Properties
+
+The Qualtrics integration allows you to define your the event properties as you require within the following constraints:
+
+- `Track` events allow you to define the `properties` object
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/qualtrics/index.md:45]:
```suggestion
- Use `Track` events to define the `properties` object
```

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md [hunk 10]
```diff
@@ -1,4 +1,56 @@
...
+- `Identify` events allow you to define the `traits` object
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/qualtrics/index.md:46]:
```suggestion
- Use `Identify` events to define the `traits` object
```

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md (hunk 3 above)
Comment (rchinn-segment) [reply src/connections/sources/catalog/cloud-apps/qualtrics/index.md:14]: Passive > Active voice

File: src/connections/sources/catalog/cloud-apps/qualtrics/index.md (hunk 5 above)
Comment (rchinn-segment) [reply src/connections/sources/catalog/cloud-apps/qualtrics/index.md:24]: we > Segment and eg. > for example

**Review by rchinn-segment — CHANGES_REQUESTED** [review]
Thanks for submitting, @carlmlee! Just a few requested changes to align this with our style guide.

**Review by rchinn-segment — APPROVED** [review]
Looks good, @tcgilbert - Is there a specific date that we want to merge this? If so, I'll update the pr.

**Review by tcgilbert — APPROVED** [review]
looks good!

**Pull request merged**